src/hAPI/README.md

```diff
@@ -1,5 +1,54 @@
-# hAPI TypeScript Library 
+# What is this?
 
-## Acknowledgements 
+This repository contains the necessary components to run the multimodal audio-haptic photo renderer in the IMAGE extension using the Haply 2DIY device.
 
-The [hAPI library](https://github.com/Shared-Reality-Lab/IMAGE-browser/tree/main/src/hAPI) used in the project is a TypesSript port of the [Java hAPI library](https://gitlab.com/Haply/hAPI) originally authored by Oliver Anthony, [Haply Robotics](https://haply.co/). 
+## Requirements
+
+* A Haply 2DIY with its firmware installed.
+## How To Use
+
+* Ensure the Docker container hosting the appropriate haptic handler is running on the development / main server.
+* Connect the Haply 2DIY device through a serial USB connection.
+* In the extension settings under the "Advanced Options", turn on Developer Mode and select the "Haply 2diy" option. Change the server URL if needed.
+* Get an IMAGE rendering of a photograph, wait for a response and then and check if the list of available renderers in the pop-up window contains a photo audio-haptic renderer. If not, recheck the first three steps, and/or try obtaining an IMAGE rendering for another photograph.
+* Click the "Connect To Haply" button and select the COM port for the Haply device. Steadily grasp the 2DIY's knob and click the "START" button to initiate the audio-haptic experience.
```

Comment:
Just a note that "Connect to Haply" should probably be changed in the actual implementation. You're connecting to a 2diy, not to Haply the company.

Reply:
Note: COM port is Windows specific.

Reply:
removed the 'COM' portion in the text. We'll make a new PR for removing the 'Haply' text in the implementation.